src/api/instance/localInstance.js

```diff
@@ -256,15 +306,10 @@ export class LocalInstance implements Instance {
     };
   }
 
-  async writeLedger(ledger: Ledger): Promise<void> {
-    const ledgerPath = pathJoin(...LEDGER_PATH);
-    await this._storage.set(ledgerPath, encode(ledger.serialize()));
-  }
-
   async writeCredGraph(credGraph: CredGraph): Promise<void> {
     const cgJson = stringify(credGraph.toJSON());
     const outputPath = pathJoin(...CREDGRAPH_PATH);
-    await this._zipStorage.set(outputPath, encode(cgJson));
+    return this._zipStorage.set(outputPath, encode(cgJson));
```

Comment:
Why return a `void`.

Reply:
it's a promise, so that it can be properly awaited or paralellized by the caller.